wiki/entry_schmitz.qmd

```diff
@@ -0,0 +1,45 @@
+
+Relationship
+
+Wie auch dieser Artikel betont, ist es wichtig, dass Familien Regeln für die Nutzung der (online) Medien aufstellen und Pläne für eine sichere Nutzung erstellen. Diesem Ansatz stimmen auch weitere Forscher zu, die fordern, dass Eltern bei der Nutzung der Medien einen Fokus darauflegen, dass ihre Kinder die Medien dafür nutzen, um sich mit Freunden auszutauschen, lehrreiche Inhalte anzuschauen und sich kreativ auszuleben (Coyne et al, 2017). Viele Forschern verlangen, dass die Eltern für ihre Kindern eine Screen Time festlegen, die nicht überschritten werden soll. Man möchte so dem Abnehmen der Aufmerksamkeitsspanne entgegenwirken, welche man bei vielen Kindern und Jugendlichen beobachten konnte. Des Weiteren erhofft man sich, dass auf diese Weise vermehrt reale soziale Interaktionen gefördert werden (Nikken & Schols, 2015). 
+
+Questions
+
+1.	Was sind mögliche Langzeitfolgen?
+2.	Schadet eine hohe Nutzungsdauer den sozialen und zwischenmenschlichen Kompetenzen des Kindes?
+
+Summary 
+Da heutzutage die online Mediennutzung immer weiter individualisiert wird, fällt es vor allem jungen Nutzern immer schwerer die Grenze zwischen Online-Kommunikation und Entertainment zu finden. Da vor allem junge Menschen das Internet und die sozialen Medien häufig nutzen, ist es wichtig auf die verschiedenen Risiken für die verschiedenen Altersgruppen hinzuweisen. Je nach Persönlichkeit des Kindes oder Jugendlichen, den sozialen Umständen und des Alters kann die Mediennutzung im Internet verschiedene Risiken mit sich bringen, über die vor allem auch die Eltern ihre Kinder aufklären sollten. Diese Risiken können gesundheitliche und mentale Folgen haben. Gefahren des Internets stellen unter anderem Cyber-Mobbing, Sexting oder Kinderpornographie dar. Kinder sollten daher bereits im jungen Alter über die Risiken des Internets aufgeklärt werden und ihre Eltern sollten mit gutem Beispiel voran gehen. Jedoch gibt es auch positive Aspekte des Internets und der digitalen Mediennutzung. Lernprogramme oder aufklärende Fernsehsendungen können Kinder ab fünf Jahren dabei helfen, sich weiterzubilden und neue Dinge zu lernen. Außerdem bringen die verschiedenen Formen der Interaktion den Kindern spielerisch den Umgang mit Technologien bei. All dies ersetzt jedoch nicht die persönliche Interaktion zwischen Kindern und anderen Personen, die vor allem für Kleinkinder in ihrer Entwicklung von sehr großer Bedeutung ist und daher unter keinen Umständen vernachlässigt werden sollte. Die Autoren des Textes schlagen deswegen vor, dass Familien einen Plan erstellen, auf welche Art und Weise und wie lange die (sozialen) Medien genutzt werden sollten. Eltern sollten ihren Kindern auch Grenzen setzen, wie oft und wie lange die Medien genutzt werden dürfen.
```

Comment:
Nowadays, as online media use is becoming more and more individualized, young users in particular are finding it increasingly difficult to find the line between online communication and entertainment. Since young people in particular use the Internet and social media frequently, it is important to point out the different risks for different age groups. Depending on the personality of the child or young person, the social circumstances and the age, media use on the Internet can entail various risks, about which parents in particular should also educate their children. These risks can have health and mental consequences. Dangers of the Internet include cyber bullying, sexting or child pornography. Children should therefore be educated about the risks of the Internet at a young age and their parents should set a good example. However, there are also positive aspects of the Internet and digital media use. Learning programs or educational TV shows can help children as young as five to continue their education and learn new things. In addition, the various forms of interaction teach children how to use technologies in a playful way. However, all of this does not replace face-to-face interaction between children and other people, which is very important, especially for young children in their development, and should therefore not be neglected under any circumstances. The authors of the text therefore suggest that families make a plan on how and for how long the (social) media should be used. Parents should also set limits for their children on how often and how long the media may be used.

Reply:
Q2: In my opinion, a longer screen time can indeed have a negative impact on face-to-face interactions between children. The more time they spend on their phones, the less they get to know how to interact with other people their age. They risk not learning enough social skills which could have serious consequences on their behaviour as an adult. Being a child should be about discovering the world and the people around them through their own experiences. If they stay inside on their phone they miss out on real-life friendships and get used to a digital world rather than the real one.